PIPS/pip-009.md

```diff
@@ -0,0 +1,58 @@
+    PIP: 009
+    Title: Investor Referral Reward Program (aka Hustler PIP)
+    Author: <KRostislav Grubi>
+    Created: <2018-03-18>
+    Requires: PIP002
+
+
+## Simple Summary
+Bring in new customers (investments) to the project and receive your cut as a reward. That can help us stay independent. 
```

Comment:
> That can help us stay independent. 

How can we receive investments and stay independent? Can you explain, please?

Every investor has some expectations on ROI — otherwise why to invest? If so, what happens if we in our independency decide to do something contrary to her expectations?

Reply:
On behalf on Ross:

In traditional business investors have shares/equities after they invested in some project/company. If they have equity - there is direct dependency and they can push if they believe its needed. 

In blockchain industry it works kinda other way. Investors don't buy any equity or a share of a company (in our case we don't even have a company and they just cant have an equity). If they believe in our technology and our intentions/plans - they buy our tokens. 

So their expectations on ROI are based on our token and its market growth. We cant say this is direct a dependency between us and investors - because all they can do - buy/sell the tokens. That's all basically. They cant change our team, they cant dictate us what we need to do and when, they cant force us to change the roadmap and they cant bring their guys to our team. I would say its not a dependency, first of all its a question of trust, reputation and belief in project intentions.

So blockchain industry gives a good answer to "how we can receive investments and stay independent?" right now. I don't know if its good or not, but thats how it works now in reality. 

Overall I believe its very risky for traditional investors to invest in blockchain projects (because they just cant defend/protect their investment here like they can in traditional IPO and traditional business). But yeah, its risky but it can be muuuuch more profitable for them than invest to traditional markets (non-blockchain).

Reply:
I disagree with this out of 3 points:
- you limit your argument to token sale, then it should clearly state that only token sale are concerned by this PIP, and the token is assumed to be a utility token (not security).
- any token holder with significant share has influence over the project by affecting the market price. This can prevent us from market making for years and affect security of the chain.
- having investors prevents the project from applying for grants, so it limits the degrees of freedom.